src/Streamly/Internal/Data/Parser/ParserK/Type.hs

```diff
@@ -492,3 +378,11 @@ instance Monad m => MonadPlus (Parser m a) where
 
     {-# INLINE mplus #-}
     mplus = (<|>)
+
+{-# INLINE_LATE toParserK #-}
+toParserK :: Parser m a b -> Parser m a b
+toParserK = id
+
+{-# INLINE_LATE fromParserK #-}
+fromParserK :: Parser m a b -> Parser m a b
+fromParserK = id
```

Comment:
Should have a newline at the EOF.

Reply:
What is the point of these definitions? We have moved these out of this file, why are we still keeping these as `id`?